agir/groups/admin/panels.py

```diff
@@ -96,6 +96,33 @@ def queryset(self, request, queryset):
             )
 
 
+class TooMuchMembersFilter(admin.SimpleListFilter):
+    MEMBERS_LIMIT = 30
```

Comment:
Il y a déjà une constante définie sur le modèle `SupportGroup` :
```suggestion
    MEMBERS_LIMIT = models.SupportGroup.MEMBERSHIP_LIMIT
```